.github/labeler.yml

```diff
@@ -11,7 +11,19 @@ chore:
   - all:
     - changed-files:
       - any-glob-to-any-file: ['.github/**/*', 'hack/**/*', '*']
-      - all-globs-to-all-files: ['!*.md']
+      - all-globs-to-all-files: ['!*.md', '!go.mod', '!go.sum', '!*.go']
+
+github_actions:
+  - changed-files:
+    - any-glob-to-any-file: ['.github/**/*.yml', '.github/**/*.yaml', '.github/**/*.md']
+
+go:
+  - changed-files:
+    - any-glob-to-any-file: ['**/*.go', 'go.mod', 'go.sum']
+
+python:
+  - changed-files:
+    - any-glob-to-any-file: ['**/*.py', '**/requirements.txt']
```

Comment:
tests use some yaml files and secrets as well

Reply:
are they `python` changes or fall under the `test` label?

Reply:
technically both, are we excluding tests from python?